lib/index.js

```diff
@@ -97,8 +121,9 @@ function makeHandleRequest(logger, app, routes) {
     return function (req, res) {
         const params = getParams(req, routes);
         logJson(logger, params);
+
         app.proxyRouter(params, {
-            done: makeHandleResponse(logger, res)
+            done: makeHandleResponse(logger, res, parseContentHandling(params, app))
```

Comment:
This might be a little clearer if you did `const convertToBinary = parseContentHandling(params, app)` above then passed that in.  I.e. it'd be easier to know without looking at `makeHandleResponse` that it's a boolean flag, since `parse` doesn't give a strong hint about what gets returned.